components/Search/forms/SearchCasesForm.jsx

```diff
@@ -120,6 +122,26 @@ const SearchCasesForm = ({ onFormSubmit, defaultValues }) => {
       </div>
 
       <Button label="Search" type="submit" disabled={!isDirty} />
+      <span
+        className="govuk-body"
+        onClick={() => {
+          reset({
+            start_date: '',
+            end_date: '',
+          });
+        }}
+        style={{
+          background: 'transparent',
+          color: '#00664f ',
+          textDecoration: 'underline',
+          border: 'none',
+          cursor: 'pointer',
+          marginLeft: '20px',
+          marginTop: '10px',
```

Comment:
We don't need most of these rules, we can simplify it as:
```suggestion
              margin-left: 1rem;
              line-height: 2.5rem;
```